mixins/form.jsx

```diff
@@ -342,13 +347,23 @@ module.exports = {
     if (!valid) {
       return;
     }
+    var description = this.getIntlMessage("mozilla_donation");
+    var handlerDesc = "";
     submitProps = this.buildProps(props);
+    if (submitProps.frequency === "monthly") {
+      description = this.getIntlMessage("mozilla_monthly_donation");
+      handlerDesc = this.getIntlMessage("monthly");
```

Comment:
Display in the strpe widget that it's monthly, just as a handy reminder. Single just displays nothing, because I think it's safe to assume it's implied.